src/XMLimport.h

```diff
@@ -50,7 +50,7 @@ class XMLimport : public QXmlStreamReader
 public:
     XMLimport(Host*);
 
-    bool importPackage(QIODevice* device, QString packageName = QString(), int moduleFlag = 0);
+    bool importPackage(QIODevice*, QString packageName = QString(), int moduleFlag = 0, QString* pVersionString = Q_NULLPTR);
```

Comment:
Well I was going to stay quiet but can I whisper the following as well: `Q_DECL_OVERRIDE`, possibly `Q_DECL_FINAL` and, for debug output I vaguely recall a discussion about identifying the function in which an error occurs, so `Q_FUNC_INFO()` might be useful.  {I tend to manually enter the function and possibly argument types in my qDebug() lines but this may help to do it auto-magically}!